features/lwipstack/LWIPStack.h

```diff
@@ -341,7 +341,7 @@ class LWIP : public OnboardNetworkStack, private mbed::NonCopyable<LWIP> {
     nsapi_error_t add_dns_server(const SocketAddress &address, const char *interface_name) override;
 
     /** @copydoc NetworkStack::get_ip_address */
-    virtual nsapi_error_t get_ip_address(SocketAddress *address);
+    virtual nsapi_error_t get_ip_address(SocketAddress *address) override;
```

Comment:
Style preference is to drop the `virtual` - only use one of `virtual`, `override`, `final`. (Even though each implies the previous). https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#Rh-override

That's the way the rest of the file is written.

Reply:
Thanks. [Done](https://github.com/ARMmbed/mbed-os/compare/940063bfc45345af988acc2d06f2622a4803856a..1beb8d3810cf8b479513ac19dd46cc029d25c1a7).